### ISSUE TITLE
Visual Accessibility Issue; Filter Conversion / Updating Notifications.

### ISSUE THREAD

**Issue by ghost** [AdguardTeam/AdguardForiOS#1502]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

- **AdGuard version:**
  - AdGuard for iOS v4.0.0
- **Device model and storage size:**
  - iPad 6th Generation 128b
- **Operating system and version:**
  - 13.4 β

### Screenshots

<details><summary>Screenshot:</summary>

![61E0F3CA-0800-493B-88FD-D5BA32A5A289](https://user-images.githubusercontent.com/49786450/74547831-bacd9280-4f1a-11ea-947b-087052bec947.png)
![584F2681-A65E-4D0F-BD23-C2C05CAC8889](https://user-images.githubusercontent.com/49786450/74547834-bc975600-4f1a-11ea-90ea-e45ce20f402e.png)

</details>

### Additional Information

  As seen at the bottom of the two screenshots, the information that the lists are being converted & that the lists are being updated, I am submitting this as a bug and not a feature request as accessibility isn't a feature, it's expected by not only those with, in this case, a visual disability that would prohibit them from either seeing the notices at all and/or being able to read what they say, but also by disability rights activists, abled or disabled, and a growing portion of the general able bodied population.
  The size is an obvious issue for accessibility; though the contrast is also an issue. The black lettering on the gray background is not of a high enough contrast to make it accessible. Guides you may find helpful are:

1. https://webaim.org/standards/wcag/checklist#sc1.4.3
2. https://www.w3.org/TR/WCAG21/

  There is a minimum contrast level that is explained for various sizes, types, and purposes of text & there is also an enhanced level explained in the same ways. Basically, the minimum accessible contrast is ~4.5:1 and the enhanced contrast level is ~7:1 (excepting large scale text for each, which lowers the contrast level required for accessibility).

 

  As a final statement on this issue, even if the AdGuard team does not consider accessibility important or that accessibility is only an afterthought and is not considered during the design of UIs, being able to explicitly tell the public that as a company you are working toward designing accessible products *so that everyone may use*, and by default, **purchase your products**, is a worthy goal, even if only for the bottom line as the results are the same no matter the motivation.

**Comment by zznosar** [comment]
Isn’t this option not enabled by default and can only be seen when developer mode is on?

I believe this should be an enhancement not a bug.

First is, this is disabled by default.
Second is, this does not impact the feature of the app.
Third is, this is just and additional FYI to the user of what is happening in the background.

**Comment by AlikhanAliev** [comment]
You're both right. It's a developers option hidden by default but being friendly with people who have a visual disability is really important. So, we'll think about it, thank you!

**Comment by mozdevcontrib** [comment]
@zznosar You're correct. But "Developer Mode" reveals the settings shown in this screenshot -

![343C5D6F-C22E-40E8-AC3F-2CB24906D463](https://user-images.githubusercontent.com/49786450/74678018-cec9fc00-5187-11ea-8d4d-6237749b7d46.jpeg)

As you can see, settings such as using simplified filters and choosing the tunnel type to be used are reached through enabling Developer Mode. Both of these settings are, I believe, generally considered as normal settings to not/use as desired by the user. 

@AlikhanAliev 

Thank-you.

And thank-you to the AdGuard team for marking this as Enhancement:P3 Medium:UI. 👍🏼
